otter/test/convergence/test_service.py

```diff
@@ -632,7 +632,7 @@ def test_returns_retry(self):
         def plan(*args, **kwargs):
             return pbag([
                 TestStep(Effect(Constant((StepResult.SUCCESS, [])))),
-                ConvergeLater(),
+                ConvergeLater(reasons=freeze(['mywish'])),
```

Comment:
Non-blocking - are reasons supposed to be a pyrsistent data structure?  If so, should that be in the ConvergeLater attributes constructor?